cirq/circuits/circuit.py

```diff
@@ -1516,6 +1517,10 @@ def _draw_moment_in_diagram(moment: Moment,
                             precision: Optional[int],
                             moment_groups: List[Tuple[int, int]]):
     x0 = out_diagram.width()
+
+    if not moment.operations:
+        out_diagram.write(x0, 0, '─')
```

Comment:
Does the empty string also work? This implementation will do the wrong thing for ascii-only diagrams.